pkg/operator/operator.go

```diff
@@ -210,6 +221,10 @@ func LoadOperatorConf(cmd *cobra.Command) *Conf {
 		c.Deployment.Spec.Template.Spec.ImagePullSecrets =
 			[]corev1.LocalObjectReference{{Name: options.ImagePullSecret}}
 	}
+
+	c.SecurityContextConstraints.Name = c.SA.Name
+	c.SecurityContextConstraints.Namespace = options.Namespace
```

Comment:
namespace? for SCC? seems irrelevant and might cause errors.

Reply:
moved to cleanup